client/src/components/BestNearByAtt.jsx

```diff
@@ -48,8 +48,8 @@ BestNearbyAttractions.propTypes = {
   attractions: PropTypes.arrayOf(PropTypes.object).isRequired,
   parentLocation: PropTypes.shape({
     lat: PropTypes.number,
-    long: PropTypes.number,
-  }),
+    lng: PropTypes.number,
+  }).isRequired,
```

Comment:
I think you can change this to PropTypes.objectOf(PropTypes.number) without eslint minding, if you want to reduce strictness of the validation